foodsearch/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
```

Comment:
You can remove logo192.png and logo512.png from the public folder and also remove from this file.